special-interest-groups/sig-expr/README.md

```diff
@@ -0,0 +1,33 @@
+# Expression Special Interest Group(Expr SIG)
+
+Expression SIG covers the tidb expression implementations. We are focused on
+the development and maintenance of the TiDB [Function and
+Operator](https://pingcap.com/docs/dev/reference/sql/functions-and-operators/reference/)
+behavior and their dependencies, including but not limited to:
+
+* Support new features.
+* Improve the performance.
+* Fix the bugs.
+
+## Roles and Organization Management
+
+See [SIG Expr Roles and Organization Management](./roles-and-organization-management.md)
+
+## Members
+
+See [SIG Expr Member List](./member-list.md)
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Wednesdays at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: [pingcap.zoom.us/j/3190482744](https://pingcap.zoom.us/j/3190482744)
+* Meeting Notes: [Meeting Notes and Agenda](https://docs.google.com/document/d/1bHZs-qj91jH0crUFgROQ0P076rosDQNm2CsUTvDr0zo/edit)
+
+## Contact
+
+* Slack: channel #sig-expr in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```

Comment:
tidbcommunity - > TiDB Community?

Reply:
I prefer to keep the old one. It's also keep the same form of what we described in [Vectorized Expression Working Group](https://github.com/pingcap/community/blob/master/working-groups/wg-vec-expr.md)